### PR TITLE
Convert `Glossary` into an index page

### LINKED ISSUE #4804: Clean up references to Glossary
context: #3824

1. the glossary is still in place
2. a lot of redirects point to it
3. a lot of articles still point to the glossary

what needs to be done:
- [ ] redirects must point to the new articles;
- [x] direct links to the glossary must point to the new articles;
- [ ] the glossary needs to be an [index article](https://github.com/ppy/osu-wiki/blob/master/wiki/Article_styling_criteria/Formatting/en.md#index-articles)

_Originally posted by @TicClick in https://github.com/ppy/osu-wiki/issues/3824#issuecomment-762525385_

### PR DESCRIPTION
**Pull request by DoryanR**
resolve #4804

Convert Glossary page into an index page.

### CODE REVIEW

File: wiki/Glossary/en.md [hunk 1]
```diff
@@ -1,912 +1,20 @@
...
-## Z
-
-### z/x
-
-z/x are the default keys on the keyboard to [tap](#tap) in osu!, a very popular alternative to the left and right mouse buttons.
-
-During gameplay, while `Z` is pressed, the left mouse button will not work and vice versa. The same applies to the `X` key and the right mouse button and vice versa.
-
-The keyboard tapping keys can be changed in the options menu.
+Index page for all articles concerning Glossary
```
Comment (TPGPL) [inline wiki/Glossary/en.md:3]:
```suggestion
Index page for all articles concerning osu!-specific terms.
```
or something similiar; the sentence shouldnt mention the Glossary i guess

**Comment by peppy** [comment]
How does this work? What's the method of accessing all the remaining terms?

**Comment by DoryanR** [comment]
For the moment there are no solutions, but I think putting the current Glossary in `meta/unused` to keep track of it is a good idea.

**Comment by Joehuu** [comment]
> How does this work? What's the method of accessing all the remaining terms?

Via the sitemap: https://osu.ppy.sh/wiki/en/Sitemap

**Comment by peppy** [comment]
Might be worth linking there with an explanation from the glossary page, if we plan to deprecate it?

File: wiki/Glossary/en.md [hunk 2]
```diff
@@ -1,912 +1,20 @@
...
+- [Combo_multiplier_effect](Combo_multiplier_effect)
+- [Combobreak](Combobreak)
+- [Comboburst](Comboburst)
+- [Cookie](Cookie)
+- [Health_bar](Health_bar)
+- [Ignore_list](Ignore_list)
+- [Lead-in](Lead-in)
+- [Local_song_offset](Local_song_offset)
+- [Loved](Loved)
+- [Touhosu!](Touhosu!)
```
Comment (TPGPL) [inline wiki/Glossary/en.md:20]: might wanna add something like this

```suggestion
- [Touhosu!](Touhosu!)

## Trivia

- As of 2021-05-31, the Glossary has been split into seperate articles to allow further expandibility of the osu! wiki. All articles for corresponding terms are now available on the [sitemap](/wiki/Sitemap).
```

**Review by TPGPL — APPROVED** [review]

**Pull request merged**